truenas/webui
# NAS-119698 / 23.10 / Unsupported type = cron and uri when trying to Launch a docker image

Testing steps:
1. Make sure you are connected to `m40g3-137.dc1.ixsystems.net`
2. Go to APPS -> Available Applications -> Search Jacket (test_list_default)
3. Click Install 
<img width="393" alt="Screenshot 2023-01-18 at 15 48 24" src="https://user-images.githubusercontent.com/22980553/213188083-d75d7607-b156-45c1-8386-a1de3af2eb21.png">
4. At the top part of the form search for `CRON TEST` and `URI TEST`

What you will see is:
<img width="820" alt="Screenshot 2023-01-19 at 10 59 26" src="https://user-images.githubusercontent.com/22980553/213398937-a94b03d0-5fb7-43eb-914f-4352e5f3cf57.png">

Schema example:
```
        {
            "variable": "uri_test",
            "label": "URI TEST",
            "group": "General Settings",
            "schema": {
                "type": "uri"
            }
        },
        {
            "variable": "cron_test",
            "label": "CRON TEST",
            "group": "General Settings",
            "schema": {
                "type": "cron",
                "default": {
                    "hour": "*",
                    "minute": "*",
                    "month": "*",
                    "dom": "*",
                    "dow": "*"
                }
            }
        },
```
